src/details-element-polyfill/polyfill.coffee.erb

```diff
@@ -56,11 +56,37 @@ polyfillProperties = ->
           removeAttribute.call(this, name)
 
 polyfillToggle = ->
-  onTogglingClick (element) ->
+  onTogglingTrigger (element) ->
+    summary = element.querySelector "summary"
```

Comment:
I prefer using parens for function calls in CoffeeScript even though they're optional. Mind updating to follow that style throughout?